Formula/speculid.rb

```diff
@@ -0,0 +1,35 @@
+class Speculid < Formula
+  desc "Easily Build Xcode Image and App Icon Assets from Graphic Files."
+  homepage "http://www.speculid.com"
+  url "https://github.com/brightdigit/speculid/archive/1.0.0.tar.gz"
+  sha256 "929f313fd09576d4d17ea48afe8a0247fb2e788b000b5ae3bdaade71f2b2df3f"
+  head "https://github.com/brightdigit/speculid.git", :branch => "develop"
+
+  option "with-debug", "Compile Speculid with debug options enabled"
+
+  depends_on "homebrew/gui/inkscape"
```

Comment:
Can't have a dependency on `homebrew/gui` and don't really want one on Inkscape.